module-apps/apps-common/notifications/NotificationTilesModel.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "NotificationTilesModel.hpp"
+#include <Image.hpp>
+#include <Style.hpp>
+
+using namespace gui;
+
+namespace
+{
+    std::map<notifications::NotificationType, UTF8> typeToIcon{
```

Comment:
1. isn't pretty
2. shouldn't be const?
3. if it would be const? can it be constexpr - there should be nothing dynamic in known int range mapping to known string?

Reply:
Changed to constexprs